### ISSUE TITLE
1.3.6. Assignment

### ISSUE THREAD

**Issue by mserykh** [Extenza-Academy/WebDev-100_2021-Q1#20]
# Assignment 

> Credits: [Turtle Ipsum](https://github.com/Instrument/semantic-html-sample) by Instrument

### Instructions

Identify a web site that you believe is NOT accessible, and create an action plan to improve its accessibility. Your first task would be to identify this site, detail the ways that you think it is inaccessible without using analytic tools, and then put it through a Lighthouse analysis. Take the results of this analysis and outline a detailed plan with a minimum of ten points showing how the site could be improved.

### Rubric

| Criteria       | Exemplary                                                                                                                                                      | Adequate                    | Needs Improvement           |
| -------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------- | --------------------------- | --------------------------- |
| student report | includes paragraphs on how the site is inadequate, the Lighthouse report captured as a pdf, a list of ten points to improve, with details on how to improve it | missing 20% of the required | missing 50% of the required |

**Comment by mserykh** [comment]
# Analysis of improvements for a []()

## Why it is should be improved?

1.  
2.

**Comment by mserykh** [comment]
@dev-experience I will finish this assignment later as we discussed after HTML lecture
